### DISCUSSION TITLE
Daily, Weekly and Free Trial Subscription Plans

### DISCUSSION THREAD

**Discussion by Element88888** [thedevdojo/wave#155] [Q&A]
Are Daily, Weekly and Free Trial Plans possible with Wave?

**Comment by bobbyiliev** [comment]
Hi,

The billing cycle depends on the payment provider that you decide to use.

If you go with Stripe you can customize it to match your needs.

<img width="643" alt="image" src="https://github.com/user-attachments/assets/4e1f80f3-dffd-4cdd-abd5-9dc5eb4a4e94">

Best thing to do is to just spin up Wave locally and play around with it! 

\- Bobby

**Comment by waynetheisinger** [comment]
apparently Stripe offer a free trial period on all subscription levels so it should be really easy to implement - just adding an extra argument... then on the frontend the offer marketing text can mention the trial is available... https://docs.stripe.com/billing/subscriptions/trials?locale=en-GB
